downstream/modules/hub/proc-deploying-your-system-for-container-signing.adoc

```diff
@@ -19,12 +18,14 @@ As an {PlatformNameShort} user, you can now confirm if an EE/Container is alread
         automationhub_container_signing_service_script: _path/to/executable_
 
 
-. Navigate to automation hub.
+. Navigate to {HubName}.
 
-. In the navigation pane, select menu:Signature Keys[].
+. In the navigation panel, select menu:Signature Keys[].
```

Comment:
I don't see this selection in the UI. We'll need to verify whether this procedure is still valid, and if so, how is it done in the new UI?